### ISSUE TITLE
Configuration parameters

### ISSUE THREAD

**Issue by ming82871** [HengyiWang/Co-SLAM#25]
Hi!
Thanks for your excellent work! We intend to evaluate some other datasets with the code which you have provided. Therefore, we need to reconstruct the config file following your code example. We found that these parameters, especially the bounding box, learning rate, and loss weight, significantly impact rendering and reconstruction in tracking and mapping thread, even if the GT pose is given. Can you give me some suggestions or experience about the details of parameter configuration for the new dataset such as ETH3D (how to fine-tune the parameter on the new indoor dataset)?
Thanks!

**Comment by HengyiWang** [comment]
Also, please note there is a documentation that contains the details of each hyper-parameters [here](https://github.com/HengyiWang/Co-SLAM/blob/main/DOCUMENTATION.md).
